.github/workflows/ci.yaml

```diff
@@ -620,6 +620,51 @@ jobs:
           python --version
           pylint --ignore-missing-annotations=y homeassistant/components/${{ needs.info.outputs.integrations_glob }}
 
+  pylint-tests:
```

Comment:
Do we really need an additional job for it? Can we not extend the pylint job, or is the execution time that bad?

Reply:
pylint is one of the longer jobs... about 10 minutes for main and 10 minutes for tests.

Reply:
2 jobs: 8m 18s + 5m 58s

Reply:
With a single job... still running after 19 minutes
https://github.com/home-assistant/core/actions/runs/9598111761/job/26468675380?pr=120022

Reply:
Does pylint have some kind of cache?

Reply:
I was just wondering but it looks like one job takes a lot longer so we should use two jobs